### DISCUSSION TITLE
[Enhancement] Web support!

### DISCUSSION THREAD

**Discussion by Suriman** [dotnet/maui#143] [General]
For all those companies that for years have developed native applications it would be very useful to take advantage of knowledge and experience, to be able to generate a Web application from MAUI, using XAML, the MVVM pattern and C #. Blazor forces you to use Razor syntax and the component model is not reusable, so it is not the ideal choice.

**Comment by Rand-Random** [comment]
Duplicate of https://github.com/dotnet/maui/issues/62
